addOns/soap/src/test/java/org/zaproxy/zap/extension/soap/WSDLFilePassiveScanRuleTestCase.java

```diff
@@ -58,4 +59,15 @@ public void isWsdlTest()
         result = scanner.isWsdl(new HttpMessage()); /* Empty response. */
         assertFalse(result);
     }
+
+    @Test
+    public void isXmlContentType() throws IOException {
```

Comment:
`shouldNotBeWsdlWithJustTextXmlContentType`, or something like that.

Could you add tests covering the other cases?